app/models/offer.js

```diff
@@ -191,6 +191,10 @@ export default DS.Model.extend({
     return this.get("isClosed") || this.get("isCancelled");
   }),
 
+  canResumeReceiving: Ember.computed("isReceived", function() {
+    return this.get("isReceived");
+  }),
```

Comment:
```suggestion
canResumeReceiving: Ember.computed.alias("isReceived"),
```

Reply:
I intentionally added computed function for more condition in future.